### DISCUSSION TITLE
Short article where I mention autorestic

### DISCUSSION THREAD

**Discussion by nicfab** [cupcakearmy/autorestic#376] [General]
Hi,

I published a short article about `restic` and `autorestic` on my Digital Notepad.
I hope it will be helpful.
Enjoy reading.
Here is the article "[restic: excellent resource for local and cloud backup](https://notes.nicfab.eu/en/posts/restic/)".

**Comment by cupcakearmy** [comment]
Thanks for taking the time and spreading the word :) 
I've included it in the community section of the docs, hope that's fine :)
